Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Learn more https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files\r\n\r\n# dependencies\r\nnode_modules/\r\n\r\n# Expo\r\n.expo/\r\ndist/\r\nweb-build/\r\nexpo-env.d.ts\r\n\r\n# Native\r\n*.orig.*\r\n*.jks\r\n*.p8\r\n*.p12\r\n*.key\r\n*.mobileprovision\r\n\r\n# Metro\r\n.metro-health-check*\r\n\r\n# debug\r\nnpm-debug.*\r\nyarn-debug.*\r\nyarn-error.*\r\n\r\nidentifier.sqlite\r\n\r\n# macOS\r\n.DS_Store\r\n*.pem\r\n\r\n# local env files\r\n.env*.local\r\n*.env\r\n\r\n# typescript\r\n*.tsbuildinfo\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 990eb437ff390d09c7b918a61d12f731f562bc56)
+++ b/.gitignore	(date 1747321159567)
@@ -33,7 +33,7 @@
 
 # local env files
 .env*.local
-*.env
+.env
 
 # typescript
 *.tsbuildinfo
Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
deleted file mode 100644
--- a/.env	(revision 990eb437ff390d09c7b918a61d12f731f562bc56)
+++ /dev/null	(revision 990eb437ff390d09c7b918a61d12f731f562bc56)
@@ -1,1 +0,0 @@
-PUBLIC_EXPO_GOOGLE_API_KEY=AIzaSyAaXNQYfqMxouReAXMeB0u_XQwhejP8JOk
